src/platform/mt8186/include/platform/lib/memory.h

```diff
@@ -115,7 +115,7 @@
 #define HEAP_RT_COUNT64		32
 #define HEAP_RT_COUNT128	32
 #define HEAP_RT_COUNT256	32
-#define HEAP_RT_COUNT512	4
+#define HEAP_RT_COUNT512	32
```

Comment:
Fwiw, Intel has not had to make a change like this since switching RTOS to Zephyr.

Reply:
https://github.com/thesofproject/sof/blob/main/src/platform/library/include/platform/lib/memory.h#L97
is this intel's definition? still used by ipc4 build?

Btw, 4-->32 too big? do you observe around 15k bytes needed to enlarge?

Reply:
After confirming that 15k is only a small portion of the heap buffer, I am uncertain whether there may be other use cases that require a larger buffer size. Therefore, I decided to follow the solution in #7982.